templates/dashboard/category/modal_confirm_delete.html

```diff
@@ -1,51 +1,39 @@
+{% extends "dashboard/modal_confirm_delete.html" %}
```

Comment:
Let's rename this base template to `base_ modal_confirm_delete.html` to indicate that this is a generic view that should be extended.